app/perfdata.h

```diff
@@ -327,6 +327,16 @@ enum PerfEventType {
     PERF_RECORD_HEADER_MAX
 };
 
+/*
+ * Bits that can be set in attr.sample_type to request information
+ * in the overflow packets.
+ */
+enum perf_event_sample_format {
+    PERF_SAMPLE_REGS_USER			= 1U << 12,
```

Comment:
please copy the full definition from an up2date perf_event.h

Reply:
actually, please use PerfEventAttributes::SampleFormat instead

Reply:
Good, the using PerfEventAttributes::SampleFormat instead enum perf_event_sample_format is ok.
I think that the enum from perf_event.h is redundant in this case